pkg/store/prometheus.go

```diff
@@ -496,9 +499,32 @@ func (p *PrometheusStore) LabelValues(ctx context.Context, r *storepb.LabelValue
 		return &storepb.LabelValuesResponse{Values: []string{l}}, nil
 	}
 
-	vals, err := p.client.LabelValuesInGRPC(ctx, p.base, r.Label, nil, r.Start, r.End)
-	if err != nil {
-		return nil, err
+	var (
+		vals []string
+		sers []map[string]string
+		err  error
+	)
+
+	version := p.promVersion()
+	if version > "2.24" {
```

Comment:
It's better to use a library like https://github.com/blang/semver, but this seems fine.

Reply:
So no need to use this library. Right?